Quasardb.Native/qdb_api.cs

```diff
@@ -533,5 +533,19 @@ static qdb_api()
             [Out] out size_t points_count);
 
         #endregion
+
+        #region Functions specific to queries
+
+        [DllImport(DLL_NAME, CallingConvention = CALL_CONV)]
+        public static extern qdb_error qdb_query(
+            [In] qdb_handle handle,
+            [In] [MarshalAs(ALIAS_TYPE)] string query,
+            [Out] out qdb_query_result* result);
+
+        [DllImport(DLL_NAME, CallingConvention = CALL_CONV)]
+        public static extern qdb_error qdb_release(qdb_handle handle,
```

Comment:
Do you need to rewrite `qdb_release` for each possible parameter type?
There's already at least one overload at: https://github.com/bureau14/qdb-api-dotnet/pull/2/files#diff-4836a39de938e5d711cfc4df0f2fa67cR409.

Reply:
It is not strictly necessary, no. I followed the precedence from that other overload.

Reply:
If possible then, I would remove it.